appgate/openapi/parser.py

```diff
@@ -532,14 +543,27 @@ def make_attrib_maker(self, attrib_maker_config: AttribMakerConfig) -> AttribMak
                     factory=None,
                     definition=attrib_maker_config.definition,
                 )
-            elif format == "byte":
+            elif (
+                format == "byte"
+                and attrib_maker_config.definition.get("writeOnly") is True
```

Comment:
Is `attrib_maker_config.definition.get("writeOnly") is True` safe? I can't remember if the key/value in the definition are all typed properly or they remain as str

Reply:
They are not typed properly since at that point it's very meta. It's where we create new classes at runtime from the yaml spec. But I think it's safe, that field is `Optional[bool]`